src/pages/Components/RatingStars/code.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Code'
+order: 2
+---
+
+> Rating stars can be used in two ways : (1.) when filing or amending an opinion (2.) when displaying the overall score on a product sheet.
+
+Implementing rating stars :
```

Comment:
remove this, we will implement an automatic table of content soon

Reply:
MDX?

Reply:
Fixed